src/msw/dcprint.cpp

```diff
@@ -230,6 +231,13 @@ wxRect wxPrinterDCImpl::GetPaperRect() const
     return wxRect(x, y, w, h);
 }
 
+void wxPrinterDCImpl::SetFont(const wxFont& font)
+{
+    wxFont scaledFont = font;
+    if ( scaledFont.IsOk() )
+        scaledFont.WXAdjustToPPI(wxDisplay::GetStdPPI());
```

Comment:
I'm not sure why is this needed neither...

Reply:
Based on testing I found that `wxPrinterDCImpl` and `wxEnhMetaFileDCImpl` below are based on 96DPI. So the font has to be adjusted to this DPI as well.
Otherwise the font size in the print preview and print (to pdf) are too big.